packages/amplify-category-notifications/src/commands/notifications/add.ts

```diff
@@ -0,0 +1,58 @@
+import inquirer from 'inquirer';
+import { $TSContext, AmplifyCategories } from 'amplify-cli-core';
+import { printer } from 'amplify-prompts';
+import { enableChannel, getAvailableChannels, getDisabledChannels } from '../../notifications-manager';
+import { ensurePinpointApp } from '../../pinpoint-helper';
+import { writeData } from '../../multi-env-manager';
+
+export const name = 'add';
+export const alias = 'enable';
+
+/**
+ * Run function for amplify cli add
+ * @param context amplify cli context
+ * @returns updated context with notifications metadata
+ */
+export const run = async (context: $TSContext): Promise<void> => {
+  context.exeInfo = context.amplify.getProjectDetails();
+
+  const categoryMeta = context.exeInfo.amplifyMeta[AmplifyCategories.NOTIFICATIONS];
+  if (categoryMeta) {
+    const services = Object.keys(categoryMeta);
+    for (const service of services) {
+      const serviceMeta = categoryMeta[service];
+
+      if (serviceMeta.mobileHubMigrated === true) {
+        printer.error('Notifications is migrated from Mobile Hub and channels cannot be added with Amplify CLI.');
```

Comment:
In the original file, the method `return`'ed from here.
`if (serviceMeta.mobileHubMigrated === true) {
          context.print.error('Notifications is migrated from Mobile Hub and channels cannot be added with Amplify CLI.');
          return context;
        }`

Reply:
Original file: [packages/amplify-category-notifications/commands/notifications/add.js](https://github.com/aws-amplify/amplify-cli/pull/11025/files#diff-94cb8ba5d2798e8225c417d6dfe2584bdaeca4373800efab34c79f92793ced71)

Reply:
missed this, thanks!